qbraid/runtime/origin/job.py

```
@@ -0,0 +1,137 @@
+    "RUNNING": JobStatus.RUNNING,
+    "EXECUTING": JobStatus.RUNNING,
+    "FAILED": JobStatus.FAILED,
+    "ERROR": JobStatus.FAILED,
+}
+
+
+class OriginJobError(QbraidRuntimeError):
+    """Class for errors raised while processing an OriginQ job."""
+
+
+def _normalize_status(origin_status: Any) -> str:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `origin_status`

(ANN401)